site/en/docs/webstore/troubleshooting/index.md

```diff
@@ -17,6 +17,38 @@ each violation a human-readable ID. These IDs are composed of two words: a color
 example, Yellow Magnesium corresponds to the general class of errors where the extension does not
 behave as expected.
 
+## Additional requirements for Manifest V3 {: #additional-requirements-for-manifest-v3 }
+
+Corresponds to notification ID: `Blue Argon`
+
+The intent of this policy is to ensure that Manifest V3 extensions are not including remotely hosted code.
+
+### Common reasons for rejection
+
+* Including a `<script>` tag that points to a resource that is not within the extension's package.
+
+* Using JavaScript's `eval()`` method or other mechanisms to execute a string fetched from a remote source.
+
+* Building an interpreter to run complex commands fetched from a remote source, even if those commands are fetched as data.
+
+### How can you rectify this? {: #how-can-you-rectify-this_23 }
+
+* Double check all code for references to external JavaScript files, which should be replaced with internal extension files.
+
+* Review the Manifest V3 migration guide [Improve extension security](/docs/extensions/migrating/improve-security/) for a walkthrough on alternatives to execution of arbitrary strings and remotely hosted code.
```

Comment:
Future improvement: Should we eventually link to @sebastianbenz's remote code migration link?